### PR TITLE
[WiP] typo fixed

### PR DESCRIPTION
**Pull request by max123kl**

### CODE REVIEW

**Comment by jnavila** [comment]
I also spotted these typos. They need to be sent upstream (to the git project). I have others waiting. Please continue adding here and when it's settled, I'll send a patch to Git project.

Thanks

File: en/git-commit.txt [hunk 1]
```diff
@@ -307,7 +307,7 @@ The possible options are:
 	- 'normal' - Shows untracked files and directories
 	- 'all'    - Also shows individual files in untracked directories.
 
-The default can be changed using the status.showUntrackedFiles
+The default can be changed using the `status.showUntrackedFiles`
```
Comment (jnavila) [inline en/git-commit.txt:310]: Not sure about this one. The markup is not consistant, but it seems configuration variables are not back-quoted, when commands and command line options are.

**Comment by max123kl** [comment]
@jnavila 
I have noticed that in some passages the Markdown syntax is used and in other paragraphs the AsciiDoc syntax is obviously used to achieve e.g. bold or italic.

What if this is not consistent or should the parser detect this?

Example (~/en/[git-config.txt](https://github.com/jnavila/git-manpages-l10n/blob/master/en/git-config.txt#L168), line 168 and subsequent):

```
--type <type>::
  'git config' will ensure that any input or output is valid under the given ...
```
  
On [git-scm.com](https://git-scm.com/docs/git-config) the "git config" is rendered as bold, on [Debian Manpages](https://manpages.debian.org/experimental/git-man/git-config.1.en.html) it is rendered as italic.

What is the intended outcome?

**Comment by jnavila** [comment]
That's one big issue with using different toolchains with different options:

* *Debian* as well as the main Git repo are using plain old Python asciidoc
* *git-scm.com* is using asciidoctor with adhoc setup
* For MIT, i don't know their toolchain

Another point is that Asciidoctor has a "compatibility" mode which tries to mimic old Asciidoc. Git manpages are written using this older approach because the Git project is still using Asciidoc tool chain. So, the output should stick to what's available on Debian's pages.

I'm going to investigate why the output on git-scm.com does not match it.

**Comment by jnavila** [comment]
OK, it turns out that html output is :
 * `i` for Asciidoc
 * `em` for Asciidoctor. Then git-scm.com applies a stylesheet to render it as bold instead of italic.

File: en/git-commit.txt (hunk 1 above)
Comment (chizzel36) [reply en/git-commit.txt:310]: Yeah right

**Comment by jnavila** [comment]
This issue can be closed.  Some changes have already been upstreamed.

**Comment by jnavila** [comment]
I think we can close this issue. Typos in the English original text have to be fixed upstream.

**Pull request closed without merging**